config/ufs/machines/config_compilers.xml

```diff
@@ -644,6 +644,13 @@ using a fortran linker.
   <NETCDF_PATH>/apps/netcdf/4.3.0-intel</NETCDF_PATH>
 </compiler>
 
+<compiler COMPILER="intel" MACH="jet">
+  <MPICC> mpiicc  </MPICC>
+  <MPICXX> mpiicpc </MPICXX>
+  <MPIFC> mpif90 </MPIFC>
+  <NETCDF_PATH>/apps/netcdf/4.7.0/intel/18.0.5.274</NETCDF_PATH>
```

Comment:
Shouldn't this be provided by  ```<command name="load">netcdf/4.7.0</command> ```in config_machines.xml (line 475 below) Loading the module *and* hardcoding the path means that you will need to remember to change it in two places later.